### PR TITLE
Removes Zeppelin code and docs

### PR DESCRIPTION
**Pull request by derek-ho**
### Description
Removes Zeppelin code and docs

### Issues Resolved
[List any issues this PR will resolve]

### Check List
- [ ] New functionality includes testing.
  - [ ] All tests pass, including unit test, integration test and doctest
- [ ] New functionality has been documented.
  - [ ] New functionality has javadoc added
  - [ ] New functionality has user manual doc added
- [ ] Commits are signed per the DCO using --signoff

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.
For more information on following Developer Certificate of Origin and signing off your commits, please check [here](https://github.com/opensearch-project/OpenSearch/blob/main/CONTRIBUTING.md#developer-certificate-of-origin).

### CODE REVIEW

**Comment by derek-ho** [comment]
@ps48 is zeppelin still being used? If not I would be happy to change this to a PR of removal of zeppelin code and docs

**Review by YANG-DB — APPROVED** [review]

**Comment by ps48** [comment]
> @ps48 is zeppelin still being used? If not I would be happy to change this to a PR of removal of zeppelin code and docs

@derek-ho That would be great! 😄

File: public/components/notebooks/docs/dev/Zeppelin_backend_adaptor.md
Comment (Swiddis) [inline public/components/notebooks/docs/dev/Zeppelin_backend_adaptor.md:78]: If we're not going to provide specific values it can be better to not use a placeholder with brackets. Maybe "Use the username `admin` and the configured password (default `admin`)".

**Comment by derek-ho** [comment]
@ps48 @Swiddis can you take another look here? It seems to run ok on my local, but need another set of eyes here

**Review by Swiddis — APPROVED** [review]

**Review by ps48 — CHANGES_REQUESTED** [review]
@derek-ho we have some more files related to Zeppelin you may want to remove before merging this PR: 
1. `server/adaptors/notebooks/zeppelin_backend.ts`
2. `public/components/notebooks/components/helpers/zeppelin_parser.tsx`
3. `server/adaptors/notebooks/index.ts`
4. `public/components/notebooks/components/helpers/__tests__/sampleZeppelinNotebooks.tsx`

Also, it would be great to link the previously created issue for removing Zeppelin in description.

**Review by ps48 — COMMENTED** [review]
Thanks for the changes!

**Review by ps48 — APPROVED** [review]

**Pull request merged**
